src/AppInstallerRepositoryCore/Rest/Schema/1_1/Json/ManifestDeserializer_1_1.cpp

```diff
@@ -12,6 +12,30 @@ namespace AppInstaller::Repository::Rest::Schema::V1_1::Json
     {
         // Installer
         constexpr std::string_view MSStoreProductIdentifier = "MSStoreProductIdentifier"sv;
+        constexpr std::string_view ReleaseDate = "ReleaseDate"sv;
+        constexpr std::string_view InstallerAbortsTerminal = "InstallerAbortsTerminal"sv;
+        constexpr std::string_view InstallLocationRequired = "InstallLocationRequired"sv;
+        constexpr std::string_view RequireExplicitUpgrade = "RequireExplicitUpgrade"sv;
+        constexpr std::string_view UnsupportedOSArchitectures = "UnsupportedOSArchitectures"sv;
+        constexpr std::string_view AppsAndFeaturesEntries = "AppsAndFeaturesEntries"sv;
+        constexpr std::string_view DisplayName = "DisplayName"sv;
+        constexpr std::string_view Publisher = "Publisher"sv;
+        constexpr std::string_view DisplayVersion = "DisplayVersion"sv;
+        constexpr std::string_view ProductCode = "ProductCode"sv;
+        constexpr std::string_view UpgradeCode = "UpgradeCode"sv;
+        constexpr std::string_view InstallerType = "InstallerType"sv;
+        constexpr std::string_view Markets = "Markets"sv;
+        constexpr std::string_view AllowedMarkets = "AllowedMarkets"sv;
+        constexpr std::string_view ExcludedMarkets = "ExcludedMarkets"sv;
+        constexpr std::string_view ElevationRequirement = "ElevationRequirement"sv;

```

Comment:
>ElevationRequirement [](http://example.com/codeflow?start=35&length=20)

Didnt see this in the Rest Schema 1.1. Was this missed and needs to be added there?

Reply:
Yes, this(From comments above) and expected return codes(Flor's) are 2 new fields added after our v1.1 schema review meeting, we'll have to update the rest spec later.